recipe/meta.yaml

```diff
@@ -23,26 +24,17 @@ requirements:
   host:
     - python
     - pip
-    - urllib3
-    # secure
-    - pyopenssl >=0.14
-    - cryptography >=1.3.4
-    - idna >=2.0.0
-    - certifi
-    - ipaddress  # [py<=33]
-    - pysocks >=1.5.6,<2.0,!=1.5.7
+    - setuptools
+    - wheel
   run:
     - python
-    - urllib3
-    # secure
-    - pyopenssl >=0.14
-    - cryptography >=1.3.4
-    - idna >=2.0.0
-    - certifi
-    - ipaddress  # [py<=33]
-    - pysocks >=1.5.6,<2.0,!=1.5.7
+    - urllib3 >=1.23
```

Comment:
Should be `urllib3 >=1.25` https://github.com/jelmer/dulwich/blob/dulwich-0.21.3/setup.cfg#LL54C5-L54C18

Reply:
Fixed